premailer/tests/test_premailer.py

```diff
@@ -23,6 +23,32 @@
 )
 
 
+def ok_(expr, msg=None):
+    """
+    Shorthand for assert.
+
+    Copied from Nose.
+    """
+    if not expr:
+        raise AssertionError(msg)
+
+
+def eq_(a, b, msg=None):
+    """
+    Shorthand for 'assert a == b, "%r != %r" % (a, b).
+
+    Copied from Nose.
+    """
+    if not a == b:
+        raise AssertionError(msg or "%r != %r" % (a, b))
+
+
+def assert_raises(exc_class, func, *args):
+    """Compatibility with Nose API for pytest."""
+    with pytest.raises(exc_class):
+        func(*args)
```

Comment:
These provide compatibility with Nose APIs. I can replace their usage in this file with direct pytest equivalent if you prefer.